packages/common/src/store/pages/audio-rewards/slice.ts

```diff
@@ -119,6 +119,17 @@ const slice = createSlice({
             is_disbursed: true
           }
         }
+        const newlyDisbursedAmount = specifiers.reduce(
+          (acc, val) => acc + val.amount,
+          0
+        )
+        const previouslyDisbursedAmount =
+          { ...state.userChallengesOverrides[challengeId] }.disbursed_amount ??
```

Comment:
why the spread?

Reply:
supposed it could be `state.userChallengesOverrides[challengeId]?.disbursed_amount`. updating